docs/src/modules/components/MarkdownElement.js

```diff
@@ -505,6 +508,7 @@ const Root = styled('div')(
         color: `var(--muidocs-palette-primary-light, ${darkTheme.palette.primary.light})`,
       },
       '& kbd.key': {
+        color: '#fff',
```

Comment:
Fix the regression from #34976

<img width="164" alt="Screenshot 2022-11-12 at 18 15 17" src="https://user-images.githubusercontent.com/3165635/201486255-d2eda1ad-4bb6-4d89-a499-2b4bd43e64b3.png">

https://deploy-preview-34976--material-ui.netlify.app/material-ui/react-snackbar/#accessibility